### PR TITLE
Private variables in Nat directory

### PR DESCRIPTION
**Pull request by guilhermehas**
I put private variables in Nat directory to make it more concise.

### CODE REVIEW

File: src/Data/Nat/Base.agda [hunk 1]
```diff
@@ -38,6 +38,10 @@ open import Agda.Builtin.Nat public
 open import Agda.Builtin.Nat public
   using () renaming (_<_ to _<ᵇ_)
 
+private
+  variable
+    n : ℕ
```
Comment (MatthewDaggitt) [inline src/Data/Nat/Base.agda:43]: Sorry to be difficult but I think this one is a) against the style-guide (types should only be variablised if they are *used in* the definition of the main type of the file and `ℕ` isn't used, it *is* the main definition) and b) probably overkill for the number of uses, especially as 2 out of the 5 uses are going to be removed when I get round to fixing https://github.com/agda/agda-stdlib/issues/1670.

**Review by MatthewDaggitt — COMMENTED** [review]
Other than that, the others look good to me! Thanks!

**Comment by guilhermehas** [comment]
I removed my changes from this file.

**Comment by MatthewDaggitt** [comment]
Great, thanks!

**Pull request merged**